app-install-banner/deferred-banner/index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<!--
+Copyright 2015 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<html>
+  <head>
+    <title>Web App Install Banner: Defer prompt</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="/samples/styles/main.css">
+    <link rel="manifest" href="manifest.json">
+    <script>
```

Comment:
Because that is where most of the logic is and I didn't want peple to have to hunt it down.